docs/start_doc_server.sh

```diff
@@ -20,5 +20,11 @@ DOCS_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 readonly DOCS_DIR
 
 (cd "${DOCS_DIR}"/_build || exit;
-    python -m http.server 8000
+    version=$(python -V 2>&1 | grep -Po '(?<=Python )(.+)')
+    if [[ -z "$version" ]]
+    then
+        python3 -m http.server 8000
```

Comment:
I think we do not need to check the version at all. Simply using python3 wil be enough, without the need of checking the version.

Reply:
ok I should just replace the python with python3?

Reply:
yep.

Reply:
Well actually just replacing `python` with `python3 in the original command is enoug. No need to ad "version", "if" etc. 

```
python -m http.server 8000 => python3 -m http.server 8000
```

Reply:
That's it - much simpler and it will work in all cases.

Reply:
ok